### DISCUSSION TITLE
How can I use prisma in my sveltkit app on IIS server?

### DISCUSSION THREAD

**Discussion by nicholasvperry** [prisma/prisma#18908] [Q&A]
I have a sveltekit app that uses prisma. When I run it in dev mode it works just fine. When I build it locally and run
`npm run preview`
it runs just fine. However when I put it on my server with IIS I am getting the following error:

In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues at new PrismaClient (C:\wwwroot\build\node_modules.prisma\client\index.js:3:11) at file:///C:/wwwroot/build/server/chunks/_server.ts-61736086.js:5:37 at ModuleJob.run (node:internal/modules/esm/module_job:194:25)

I'm not sure what is causing this issue or what else I can include to possibly help diagnose the problem. I do not have prisma installed on my server, however I should not need to because all the files should be being passed on my build production when I am adding it to the server. The index file is loading so I know it is loading the app, it just seems as though it is not getting the prisma files.

**Comment by nurul3101** [comment]
Hello @nicholasvperry 👋 

Which version of SvelteKit are you using? 
There was a similar discussion in which SvelteKit was failing while deploying because the env variables were not read correctly.
- https://github.com/prisma/prisma/discussions/17602

Are you using the latest version of Sveltekit?

**Reply by nurul3101** [reply]
@nicholasvperry I am checking to see if you had a chance to look at my last message. Do let us know in case you need any assistance.

**Reply by nurul3101** [reply]
If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
